projects/opendr_ws/src/perception/scripts/object_detection_2d_gem.py

```diff
@@ -121,11 +174,12 @@ def __init__(self,
         # Initialize the detection estimation
         model_backbone = "resnet50"
 
-        self.gem_learner = GemLearner(backbone=model_backbone,
-                                      num_classes=7,
-                                      device=device,
-                                      )
-        self.gem_learner.fusion_method = 'sc_avg'
+        self.gem_learner = GemLearner(
+            backbone=model_backbone,
+            num_classes=7,
+            device=device,
+        )
+        self.gem_learner.fusion_method = "sc_avg"
         self.gem_learner.download(path=".", verbose=True)
 
         # Subscribers
```

Comment:
(Can't add comment directly to unchanged lines)

Both subscribers should have `queue_size=1, buff_size=10000000` arguments added to avoid delays, relevant issue #275

Reply:
Subscribers still don't have their queues set. Have you tested whether the node has delay issues? If not, then the queues are not needed.